pennylane/transforms/transpile.py

```diff
@@ -4,6 +4,7 @@
 from typing import Union, List
 import networkx as nx
 
+import pennylane as qml
```

Comment:
Maybe instead of importing all of pennylane here, we can just add it to the next line:
```suggestion
from pennylane import apply, Hamiltonian, eigvals
```

Explicitly importing just what we need has some mild performance benefits, and being explicit about what parts of the code base we depend upon may help eliminate circular dependencies.

Reply:
Tried this, and got into circular import issues. Importing PennyLane as a whole bypasses that.

Reply:
That disappointing.  Seems like we should starting being more careful in PennyLane as a whole about what depends on what.